src/objects.js

```diff
@@ -1,39 +1,72 @@
 export const createPerson = (name, age) => {
-  // your code here
+  // creates an object with the given name and age properties
+  let person = {
+    name: name,
+    age: age
+  };
+
+  return (person);
+
 };
 
 export const getName = (object) => {
-  // your code here
+  // returns the name property of the object
+  return(object["name"]);
+
 };
 
 export const getProperty = (property, object) => {
-  // your code here
+  return(object[property]);
 };
 
 export const hasProperty = (property, object) => {
-  // your code here
+  // checks whether object has given property
+  return object.hasOwnProperty(property);
 };
 
 export const isOver65 = (person) => {
-  // your code here
+  // returns true if the person is aged over 65
+  if (person['age'] > 65) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 export const getAges = (people) => {
-  // your code here
+  // returns the ages of each person in the array
+  return people.map(person => person.age);
 };
 
 export const findByName = (name, people) => {
   // your code here
+  return people.find(person => person.name === name);
 };
 
 export const findHondas = (cars) => {
   // your code here
```

Comment:
I would ensure all of these placeholders are removed - a bit tidier if people do look at examples of your work :)